### ISSUE TITLE
Running multiple inference

### ISSUE THREAD

**Issue by Pri0805** [MIC-DKFZ/nnDetection#279]
Hi, 

Is it possible to run inference using the nndet_predict / nndet_eval command on a new test set by specifying the new folder input and output paths instead of copying the new data into the imagesTs folder and risk overwriting the previous inference data in imagesTs as well test_result, test_analysis, test_predictions folders?

Thank you!

**Comment by partha-ghosh** [comment]
Hi @Pri0805,
No, nnDetection can only function with the folder structure mentioned in the README. You can only configure the path to the data folder with the `det_data` environment flag. 

In your case, you can rename the current `imagesTs` folder to something else (like `imagesTs2`) and copy your new test images to `imagesTs` or if you want to avoid copying, then you can create a symlink that points to your new test set folder. Also, to retain the current test results, you have to move them to a safe place before your run prediction for the new test set.

Best,
Partha

**Comment by Pri0805** [comment]
Hi @partha-ghosh,

We followed the above steps and ran the predict command. However in the test_predicitons file we notice that the old test data has been predicted again along with the new test data. Prior to running the command we renamed the following folders:

imagesTs, labelsTs in raw_splitted dir
test_analysis, test_results, test_predictions in consolidated dir 

Is there any other folder we are missing that is causing old test cases to be predicted as well?
